frontend/public/components/nav/admin-nav.tsx

```diff
@@ -160,7 +160,7 @@ const AdminNav = () => (
       <ResourceNSLink resource="rolebindings" name="Role Bindings" startsWith={rolebindingsStartsWith} />
       <ResourceNSLink resource="resourcequotas" name="Resource Quotas" startsWith={quotaStartsWith} />
       <ResourceNSLink resource="limitranges" name="Limit Ranges" />
-      <ResourceNSLink resource={referenceForModel(ChargebackReportModel)} name="Chargeback" required={FLAGS.CHARGEBACK} />
+      <ResourceNSLink resource={referenceForModel(ChargebackReportModel)} name="Chargeback" required={FLAGS.CAN_LIST_CHARGEBACK_REPORTS} />
```

Comment:
You need to check both flags. For cluster admin, `CAN_LIST_CHARGEBACK_REPORTS` will be true even if Chargeback is not enabled.

Reply:
Fixed